src/optimize/base_optimizer.js

```diff
@@ -273,6 +274,10 @@ export default class BaseOptimizer {
           log: this.log
         }),
 
+        new webpack.DefinePlugin({
```

Comment:
__note:__ by this time all JSON files were read, parsed and merged into single `Translation` object.